subprojects/gradle/module-dependencies-graph/src/main/kotlin/com/avito/module/internal/dependencies/DependenciesGraphBuilder.kt

```diff
@@ -60,7 +60,9 @@ internal class DependenciesGraphBuilder(
                                 dependenciesOnProjects(projectDependency.dependencyProject, ConfigurationType.Main)
                             }
                             .toSet()
-                    }.toMap()
+                    }
+                    .filter { (_, nodes) -> nodes.isNotEmpty() }
```

Comment:
Filter break correctness of the graph, and I don't understand what this filter gives

Reply:
Reverted this change